prac_05/wimbledon.py

```diff
@@ -0,0 +1,42 @@
+"""
+
+"""
+
+
+def main():
+    champions = read_champions()
+    champion_countries = read_champion_countries()
+    display_champions(champions, champion_countries)
+
+
+def read_champions():
+    champions_list = []
+    with open('wimbledon.csv', "r", encoding="utf-8-sig") as in_file:
+        for line in in_file:
+            detail_list = line.split(',')
+            champions_list.append(detail_list[2])
+            champions = {i: champions_list.count(i) for i in champions_list}
```

Comment:
Is `i` an index? https://github.com/CP1404/Starter/wiki/Programming-Patterns#for-loops-definite-iteration